cylc/flow/tests/network/test_zmq.py

```diff
@@ -39,30 +48,98 @@ def get_port_range():
 HOST = "127.0.0.1"
 
 
-def test_server_requires_valid_keys():
-    """Server should not be able to connect to host/port without valid keys."""
+def test_server_cannot_start_when_server_private_key_cannot_be_loaded():
+    """Server should not be able to start when its private key file
+    cannot be opened."""
+
+    server = ZMQSocketBase(zmq.REQ, suite="test_suite", bind=True, daemon=True)
```

Comment:
`test_suite` is actually a reg I use myself sometimes (when I've made it through foo, bar, baz), perhaps use a hash or a more unlikely name to avoid false positives in the test battery (we've had issues like this before).

```suggestion
    server = ZMQSocketBase(zmq.REQ, suite=f"test_suite-{time()}", bind=True, daemon=True)
```